main.go

```diff
@@ -222,6 +254,10 @@ func main() {
 	dbObject = connectDb()
 	defer dbObject.Close()
 
+	//apply migrations:
+	migration("up")
+	defer migration("down") //how to exit the program to test this???????????
```

Comment:
Let's not migrate down when the service stops, otherwise the database will loose all its data every time we stop the service.

Reply:
ok